### DISCUSSION TITLE
how to call view?

### DISCUSSION THREAD

**Discussion by edsondario** [scenic-views/scenic#354] [Q&A] [answered]
I did according to the gem manual, but I dont show/call view created. Then, how to call/show view created?

**Comment by derekprior** [comment, marked as answer]
I don't fully follow the question. Did you create and run a migration with a `create_view` statement? If so, that view should be created in your development database and visible in `schema.rb`. 

You can access it by creating an ActiceRecord model for that view (e.g if your view is `posts` create a `Post` model) or via direct SQL queries. 

If you provide more detail or even a sample app, I can be of more help.

**Reply by edsondario** [reply]
Thanks! I fix. 

I learned to directly invoke a vision. 
`ActiveRecord::Base.connection.execute("SELECT * FROM ActiveUser)`

I also learned using templates in ActiveRecord. 
**I make report.rb in folder models**
`class ActiveUser < ActiveRecord::Base
  def readonly?
    true
  end
end`

And finally call ActiveUser in rails db:console.

I believe it would allow showing how to use rails models in README.MD

**Reply by derekprior** [reply]
We do show how to use views in models in the README: https://github.com/scenic-views/scenic#can-i-use-this-view-to-back-a-model
